src/functions-and-arrays.js

```diff
@@ -1,73 +1,431 @@
 // Iteration #1: Find the maximum
 
+var maxOfTwoNumbers = [5, 7];
+
+function findMax(array) {
+  var max = -Infinity;
+
+  for (let i = 0; i < array.length; i++) {
+    if (array[i] > max) {
+      max = array[i];
+    }
+  }
+  return max;
+}
+
+var result = findMax([5, 7]);
+
+console.log("result :", result);
+
 // Iteration #2: Find longest word
-const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
 
+/* 
+I don't fully understand:
+You can use rest parameter (...) syntax 
+allows us to represent an indefinite number of arguments as an array.*/
+function findLongestWord(...strs) {
```

Comment:
- This function is spreading an array that is getting from a callback. This was not how it was asked in the instructions.